lua/ttt2/libraries/bodysearch.lua

```diff
@@ -34,6 +69,25 @@ function bodysearch.GetInspectConfirmMode()
 	return cvInspectConfirmMode:GetInt()
 end
 
+---
+-- Checks if a given player is allowed to take credits from a given corpse-
+-- @param Payer ply The player that tries to take credits
+-- @param Entity rag The ragdoll where the credits should be taken from
+-- @param[default=false] isLongRange Whether the search is a long range search
+-- @return boolean Returns if the player is able to take credits
+-- @realm shared
+function bodysearch.CanTakeCredits(ply, rag, isLongRange)
+	---
+	-- @realm server
+	if hook.Run("TTT2CheckFindCredits", ply, rag) == false then
```

Comment:
The comment is wrong and might be unneeded because it is defined at the end of the file, i guess?

Reply:
you're right that it is wrong but Alf's tool needs it in both places